frontend/lib/src/AppNode.ts

```diff
@@ -791,6 +798,9 @@ export class AppRoot {
       this.bottom.clearStaleNodes(currentScriptRunId, fragmentIdsThisRun) ||
       new BlockNode(this.mainScriptHash)
 
+    const appLogo =
+      this.appLogo?.scriptRunId === currentScriptRunId ? this.appLogo : null
+
```

Comment:
Feel like we can write a unit test for the appLogo being cleared and not cleared, right?